### PR TITLE
feat: allow setting a custom iframe title

### PR DESCRIPTION
**Pull request by pcorpet**
This allows for a better accessibility especially for screen readers.

See https://www.w3.org/TR/WCAG20-TECHS/H64.html.

### CODE REVIEW

File: src/YouTube.js [hunk 1]
```diff
@@ -226,6 +227,8 @@ class YouTube extends React.Component {
       else iframe.removeAttribute('width');
       if (this.props.opts && this.props.opts.height) iframe.setAttribute('height', this.props.opts.height);
       else iframe.removeAttribute('height');
+      if (typeof this.props.title === 'string') iframe.setAttribute('title', this.props.title);
+      else iframe.setAttribute('title', 'YouTube Video Player');
```
Comment (ruisaraiva19) [inline src/YouTube.js:231]: I'm not sure about this default value.
We should follow the same logic from other props by removing the title attribute when not provided.

**Review by ruisaraiva19 — CHANGES_REQUESTED** [review]

File: src/YouTube.js (hunk 1 above)
Comment (pcorpet) [reply src/YouTube.js:231]: Today the YouTube IFrame API is setting this default value so I didn't want to change the existing behavior. Here are the solutions I propose by default:
1. we drop the title (as you suggest)
2. we keep a hard coded "YouTube Video Player" (as I'm suggesting)
3. we try to get the default value set by YouTube IFrame API and avoid overriding it / restore it if we have overridden it
4. we try to be even smarter than YouTube and set the video's title when we get it in the IFrame

1 has the downside of modifying the current behavior AND making the component less accessible (we end up with an IFrame with no title).
2 has the downside of keeping the value hard coded in this library
3 and 4 makes for more complex code, especially 4 add some extra logic which breaks the purpose of react-youtube being a thin wrapper around the YouTube API.

Comment (ruisaraiva19) [reply src/YouTube.js:231]: I missed that the default is `YouTube video player` (video & player words lowercase), so I think we can move on with your suggestion.
Just update it to be `YouTube video player` exactly.

Comment (ruisaraiva19) [inline src/YouTube.js:231]:
```suggestion
      else iframe.setAttribute('title', 'YouTube video player');
```

**Review by ruisaraiva19 — CHANGES_REQUESTED** [review]

File: src/YouTube.js (hunk 1 above)
Comment (pcorpet) [reply src/YouTube.js]: Done

**Review by ruisaraiva19 — APPROVED** [review]

**Comment by ruisaraiva19** [comment]
@pcorpet please update your last commit message to follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) guidelines.

**Comment by pcorpet** [comment]
@ruisaraiva19 are you OK if I merge the two commits (with the initial message) and hard push them on the branch?

**Comment by pcorpet** [comment]
I would need to force push anyway to drop the second commit, so I'm going to do it.

**Pull request merged**